tools/cldr-apps/js/src/index.js

```diff
@@ -17,7 +17,7 @@ import * as cldrVue from "./esm/cldrVue.mjs";
 import * as cldrMonitoring from "./esm/cldrMonitoring.mjs";
 
 // fonts (see surveytool.css)
-import "@fontsource/noto-serif-np-hmong"; 
+import "@fontsource/noto-serif-np-hmong";
```

Comment:
unrelated, i may have missed the prettier call.

Reply:
this was leftover from #4507

Reply:
For Java, but not for JS, a PR is blocked if not normalized; would that be hard to implement for JS?